### DISCUSSION TITLE
Diving suits in Enemy submarines

### DISCUSSION THREAD

**Discussion by BSIW2ARNI** [FakeFishGames/Barotrauma#14395] [Bug reports]
I don't know if this supposed to be like that, but with mods in Enemy subs there are only default diving suits. On mid levels just one good shot in sub with railgun will destroy the crew, because these suits are not designed to work with such pressure and depth. How to change that? Only to contact the author?

**Comment by Regalis11** [comment]
I think you need to contact the creators of these mods.
